src/6model/reprs/P6opaque.c

```diff
@@ -840,7 +840,7 @@ static void compose(MVMThreadContext *tc, MVMSTable *st, MVMObject *info_hash) {
             /* C structure needs careful alignment. If cur_alloc_addr is not
              * aligned to align bytes (cur_alloc_addr % align), make sure it is
              * before we add the next element. */
-            align_to(&cur_alloc_addr, align);
+            align_to((MVMuint64 *)&cur_alloc_addr, align);
```

Comment:
Why not turn cur_alloc_addr into a proper MVMuint64 instead of casting here? I don't see a place where it would ever have a negative value.